metrics/graphite.go

```diff
@@ -38,9 +37,10 @@ func init() {
 
 // GraphiteConfig holds basic Graphite configuration.
 type GraphiteConfig struct {
-	Host   string
-	Port   int    `default:"2003"`
-	Prefix string `default:"allegro.executor"`
+	Host        string
+	MetricsUUID string
```

Comment:
Why did you add this parameter here?

Reply:
You are right, I can go back to using a local var.